website/docs/operator/data-recovery.md

```diff
@@ -4,4 +4,104 @@ title: Data Recovery
+the status of the recovery job is done manually by querying the database
+directly or by checking the status on the Cumulus Dashboard.
+
+The following are the parameters needed for recovery workflow:
+
+- buckets- AWS S3 bucket mapping used for Cumulus and ORCA configuration. Contains the following properties:
+  - name (Required)- Name of the S3 bucket.
+  - type (Optional)- the type of bucket - i.e. internal, public, private, protected. 
+
+  It can be set up using the following configuration.
+  ```json
+        "config": {
+          "buckets.$": "$.meta.buckets"
+        }
+    ```
+    Example:
+
+    ```json
+    "buckets": {
+            "protected": {"name": "sndbx-cumulus-protected", "type": "protected"},
+            "internal": {"name": "sndbx-cumulus-internal", "type": "internal"},
+            "private": {"name": "sndbx-cumulus-private", "type": "private"},
+            "public": {"name": "sndbx-cumulus-public", "type": "public"}
+        }
+    ```
+- fileBucketMaps- A list of dictionaries that contains details of the configured storage bucket and file regex. Contains the following properties:
+  - regex (Required)- The regex that matches the file extension type.
+  - bucket (Required))- The name of the key that points to the correct S3 bucket. Examples include public, private, protected, etc.
+  - sampleFileName (Optional)- name of a sample file having extension.
+
+  It can be set up using the following configuration.
+  ```json
+        "config": {
+          "fileBucketMaps.$": "$.meta.collection.files"
+        }
+    ```
+    Example: 
+    ```json
+        "fileBucketMaps":[
+            {
+            "regex":".*.h5$",
+            "sampleFileName":"L0A_HR_RAW_product_0010-of-0420.h5",
+            "bucket":"protected"
+            },
+            {
+            "regex":".*.cmr.xml$",
+            "sampleFileName":"L0A_HR_RAW_product_0010-of-0420.iso.xml",
+            "bucket":"protected"
+            }
+    ```
+- excludedFileExtensions (Optional)- A list of file extensions to ignore when copying files.
+  It can be set up using the following configuration.
+  ```json
+        "config": {
+          "excludedFileExtensions.$": "$.meta.collection.meta.orca.excludedFileExtensions"
+        }
+    ```
+    Example: 
+    ```json
+        "excludedFileExtensions":[
+            ".cmr",
+            ".hdf
+        ]
+    ```
+- defaultRecoveryTypeOverride (Optional)- Overrides the default restore type via a change in the collections configuration under `meta` tag as shown below. 
+  ```json
+  "meta": {
+    "defaultRecoveryTypeOverride": "Standard"
+  }
+  ```
+- defaultBucketOverride (Optional)- Overrides the default bucket to copy recovered files to.
+  ```json
+    "task_config": {
+      "defaultBucketOverride": "{$.meta.collection.meta.orca.defaultBucketOverride}"
```

Comment:
When talking about overrides, link to the section in `deployment-with-cumulus.md` that described the variable being overridden. In this case, it would be `#### ORCA Required Variables`

Reply:
Something like `Overrides the [orca_default_bucket](link) to copy recovered files to`?

Reply:
added